src/Init/Data/Fin/Basic.lean

```diff
@@ -100,7 +100,7 @@ instance : ShiftLeft (Fin n) where
 instance : ShiftRight (Fin n) where
   shiftRight := Fin.shiftRight
 
-instance : OfNat (Fin (no_index (n+1))) i where
+instance ofNatInst : OfNat (Fin (no_index (n+1))) i where
```

Comment:
In mathlib, we have been using the convention of naming instances like this `instOfNat` as opposed to `ofNatInst`, which I believe is also consistent with the automatic naming rules. Is this departure from the convention an indication that the automatic naming rules are going to change soon?

Reply:
Changed to `instOfNat`.